src/Build/Evaluation/LazyItemEvaluator.cs

```diff
@@ -545,6 +545,12 @@ private RemoveOperation BuildRemoveOperation(string rootDirectory, ProjectItemEl
                 }
             }
 
+            operationBuilder.MatchOnMetadataOptions = MatchOnMetadataOptions.CaseSensitive;
+            if (Enum.TryParse(itemElement.MatchOnMetadataOptions, out MatchOnMetadataOptions options))
```

Comment:
nit:
You can save one line and perhaps improve clarity slightly if you make it:
```
if (!Enum.TryParse(itemElement.MatchOnMetadataOptions, out operationBuilder.MatchOnMetadataOptions)
{
    operationBuilder.MatchOnMetadataOptions = MatchOnMetadataOptions.CaseSensitive;
}
```

I would read that as "if we try to parse it and fail, this is the default," which is clearer than "here's the default; now let's try to parse it."

Reply:
Good thought but that [doesn't compile](https://stackoverflow.com/a/4518973/12323206), fields can't be passed as out params.